saltproc/process.py

```diff
@@ -72,44 +78,60 @@ def check_mass_conservation(self):
         out_stream = self.outflow + self.waste_stream
         np.testing.assert_array_equal(out_stream, self.inflow)
 
-    def rem_elements(self, inflow):
-        """Updates Materialflow object `inflow` after removal target isotopes
-        with specific efficiency in single component of fuel reprocessing
-        system and returns waste stream Materialflow object.
+    def process_material(self, inflow):
+        """Updates :class:`Materialflow` object `inflow` by removing target
+        nuclides with specific efficiencies in single component of fuel
+        reprocessing system and returns waste stream Materialflow object.
 
         Parameters
         ----------
         inflow : Materialflow obj
-            Target material stream to remove poisons from.
+            Material flowing into the processing system component.
 
         Returns
         -------
-        Materialflow object
+        waste_stream : Materialflow object
             Waste stream from the reprocessing system component.
+        thru_flow : Materialflow object
+            Remaining material flow that will pass through the
+            reprocessing system component.
 
         """
         waste_nucvec = {}
-        out_nucvec = {}
+        thru_nucvec = {}
         # print("Xe concentration in inflow before % f g" % inflow['Xe136'])
         # print("Current time %f" % (t))
```

Comment:
Are these print statements erroneous?

Reply:
@yardasol

Reply:
I'll look into it

Reply:
Hmm, they aren't erroneous, but I don't think they need to be here. i'm okay with purging them. Alternatively, I could leave them as-is for now and create a verbose command line option that would allow a user to specify that they want this extra infro printed.

Reply:
@samgdotson thoughts?

Reply:
I see your options being:
1. uncomment them
2. uncomment them and make an issue to add a verbosity feature
3. delete them

It doesn't matter to me which option you choose, but leaving commented print statements in your code doesn't seem like good practice (I am also guilty of this).

Reply:
Agreed. I'll do option 2.